yt_dlp/extractor/bilibili.py

```diff
@@ -165,14 +164,18 @@ def _sign_wbi(self, params, video_id):
         params['w_rid'] = hashlib.md5(f'{query}{self._get_wbi_key(video_id)}'.encode()).hexdigest()
         return params
 
-    def _download_playinfo(self, bvid, cid, headers=None, qn=None):
+    def _download_playinfo(self, bvid, cid, headers=None, fatal=True, qn=None):
```

Comment:
let's not change the order of the args just in case
```suggestion
    def _download_playinfo(self, bvid, cid, headers=None, qn=None, fatal=True):
```